lib/sepa.js

```diff
@@ -778,6 +778,8 @@
       message = 'debitor country is too long.';
     }
 
+    assert_name(name, pullFrom);
```

Comment:
you should return an InvalidSupplierError if you do the error message will be explicitely shown to the user in spendesk